cmd/kubeadm/app/cmd/certs_test.go

```diff
@@ -379,14 +379,14 @@ func TestGenCSRConfig(t *testing.T) {
 	// A minimal kubeadm config with just enough values to avoid triggering
 	// auto-detection of config values at runtime.
 	var kubeadmConfig = `
-apiVersion: kubeadm.k8s.io/v1beta2
+apiVersion: kubeadm.k8s.io/v1beta3
```

Comment:
Not for this PR, but we should really make all the sample files (here and in other files) to use `kubeadmapiv1.SchemeGroupVersion.Version` so the next release bump will be simpler.
@neolit123 opinions?
@RA489 would you like to take a shot at this?

Reply:
logged issue: https://github.com/kubernetes/kubeadm/issues/2453